src/docs/product/integrations/integration-platform/index.mdx

```diff
@@ -275,105 +285,63 @@ For an example of how to build an internal integration, see [our Round Robin Iss
 
 Creating an internal integration will automatically install it on your organization.
 
-#### Auth Tokens
+### Auth Tokens
 
-Auth Tokens are what the Integration Platform calls tokens. Auth Tokens are passed using an auth header and are used to authenticate as a user account with the API. In the case of the Integration Platform, these tokens are used to authenticate as an application (with access to a specific organization) instead of a user.
+Auth Tokens are referred to as _tokens_ in the Integration Platform. Auth tokens are passed using an auth header and are used to authenticate as a user account with the API. In the case of the Integration Platform, these tokens are used to authenticate as an application (with access to a specific organization) instead of a user.
 
-When you create an Internal Integration, a token is automatically generated. Should you need multiple, or you need to swap it out, you can go into your Developer Settings > Your Internal Integration and do so.
+When you create an internal integration, a token is automatically generated. Should you need multiple tokens, or you need to get a new one, you can go to **Settings > Developer Settings > [Your Internal Integration]** and do so.
 
-You can have up to 20 tokens at a time for any given internal integration.
+You can have up to 20 tokens at a time for each internal integration.
 
-#### How to use for requests
+### Using Tokens for Requests
 
-When making requests to the Sentry API, you use the token just like you would when you're typically making [API requests](/api/auth/). Tokens are associated with the Sentry organization that created the integration (and therefore was automatically installed).
+When making requests to the Sentry API, you use the token just like you would when you're typically making [API requests](/api/auth/). Tokens are associated with the Sentry organization that created the integration and are automatically installed for that org.
 
-#### Expiration
+### Token Expiration
 
 Tokens never expire, but you can manually revoke them.
```

Comment:
In case someone searches for this, it's good to be specific here since this answer is different for public integrations. 

```suggestion
Tokens for internal integrations never expire, but you can manually revoke them.
```